geniza/corpus/views.py

```diff
@@ -530,7 +530,7 @@ def get(self, request, *args, **kwargs):
         digital_editions = document.digital_editions()
         # while sync transcription is in transition, we need to check for
         # html transcription content
-        digital_editions = [de for de in digital_editions if "html" in de.content]
+        digital_editions = [de for de in digital_editions if de.content_html]
```

Comment:
I don't think we need this check anymore, digital edition filter should be sufficient